_posts/en/newsletters/2019-10-02-newsletter.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Bitcoin Optech Newsletter #66'
+permalink: /en/newsletters/2019/10/02/
+name: 2019-10-02-newsletter
+slug: 2019-10-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter notes a proposed BIP that would allow nodes to
+implement the erlay transaction relay protocol, announces full
```

Comment:
nit regarding "erlay" here, and the two other cases in this newsletter: the authors call it "Erlay" in their paper last June (https://arxiv.org/pdf/1905.10518.pdf) and git grepping this repo returns only "Erlay" and no other instances of "erlay"